config/config.go

```diff
@@ -67,11 +67,28 @@ type (
 		MuxTransportName string `yaml:"mux"`
 	}
 
+	DescribeClusterResponseOverrides struct {
+		FailoverVersionIncrement *int64 `yaml:"failover_version_increment,omitempty"`
+	}
+
+	DescribeClusterOverride struct {
+		Response DescribeClusterResponseOverrides `yaml:"response"`
```

Comment:
Should we support a cluster name here? e.g. only modify failover if cluster name matches? Or, set different FVI values for different clusters?

(🤷 Not needed. We can get by without it)

Reply:
potentially. the setup can be expended.